### ISSUE TITLE
Icon request: icon-note

### ISSUE THREAD

**Issue by davegandy** [FortAwesome/Font-Awesome#896]
Separating out from #16.

**Comment by G-Driver** [comment]
+1

**Comment by gooeyideas** [comment]
This has been open for over 6 years. Multiple related requests and several upvotes... I think it is hilarious that you created a middle finger icon before you created a notebook icon. Why no love for the notebook?

**Comment by tagliala** [comment]
@gooeyideas our [Icon Leaderboard](https://fontawesome.com/community/leaderboard/new) contains a lot of information on how requests work

Just scroll down the Top 50!

**Comment by gooeyideas** [comment]
I get exactly how the leaderboard works and it does not favor old icon requests due to the way the issues are sorted. I posted a messenge today in hopes of generating more traffic to this issue but I believe this icon request is too old and vaguely worded for people to rally behind it! I mean look at the icon requests that you have closed in reference to this this one: Clipboards, Check Lists, Calendar Check, Lined Paper, Sticky Notes, Notebooks. Of the closed issues related to this issue the only requests that are not currently represented in Font awesome is a Notebook... 

I created a new issue #14474 for a spiral notebook which is more specific and might have done better on the leaderboard but you closed it to keep this cause open. Personally, I feel that it is silly to keep this case open. Since most of the items referenced here are complete why not close this issue break out request a request for a notebook/journal/diary icon?
